tests/misc/ExceptionsTest.py

```diff
@@ -11,6 +11,6 @@ def test_get_exitcode(self):
         self.assertEqual(get_exitcode(AssertionError()), 255)
         self.assertEqual(get_exitcode(SystemExit(999)), 999)
         self.assertEqual(get_exitcode(VersionConflict(
-            'libclang-py3 0.3', 'libclang-py3==0.2')), 13)
+            'version_1', 'version_2')), 13)
```

Comment:
these are not valid version strings.

Reply:
Actually it means that there are two different versions, that's why I put `version_1` and `version_2`.